libs/cart/testing/src/in-memory-backend/cart-root.service.integration.spec.ts

```diff
@@ -221,7 +221,7 @@ describe('DaffInMemoryBackendCartRootService | Integration', () => {
 
     beforeEach(done => {
       mockCart.coupons = [];
-      httpClient.post<any>('commands/resetDb', {carts: [mockCart]}).subscribe(() => done());
+      httpClient.post<any>('commands/resetDb', {carts: [mockCart]});
```

Comment:
I still can't tell for sure if the test is really accomplishing what it's supposed to. Maybe it would make more sense to try adding a new coupon instead of removing and adding the old one, because it isn't easy to verify that the removal actually happens the way you'd expect?

Reply:
fair enough, I have changed that spec to add a brand new coupon. I also changed another expectation to be more explicit.